src/liedle/script.js

```diff
@@ -153,4 +163,33 @@ function colourLetterFalsely(tile) {
 	tile.dataset.type = type
 }
 
+function checkEndGame(word) {
+	if (word === secretWord) {
+		handleEndGame(/*isWin =*/ true)
```

Comment:
Is the comment missing here? The rest of the code looks good.

Reply:
I added it in there as it's not clear what `true` represents. I.e. you can't tell from the name of the function `handleEndGame` and the boolean isn't saved to a variable prior to its usage.